CodeForBaltimore/Bmore-Responsive
# Issue/199/bulk emails

# Description

add additional filtering to the send endpoint for contactIds and entityIds
This piggy backs off of PR #319 and includes all of the code in that PR, so please make sure that is merged in before reviewing this one.

## Related PRs



| branch          | PR       |
| --------------- | -------- |
| Bulk Email Enhancements | #319  |

## Todos


## Deploy Notes

Notes regarding deployment the contained body of work. 

## Steps to Test or Reproduce

Go to the dashboard on healthcare rollcall
Check one or more checkboxes and then click the "Email Selected Facilities" button.

## Impacted Areas in Application

Sending bulk emails
